packages/wonder-blocks-dropdown/components/multi-select-menu.md

```diff
@@ -0,0 +1,241 @@
+The multi select menu allows the selection of multiple items. Clients are
+responsible for keeping track of the selected items in the menu.
+
+TODO(sophie): keyboard navigation and dismissal
+
+The first menu starts with nothing selected and has no selection shortcuts.
+There is also a disabled item, and this menu has a set width.
```

Comment:
One of the examples should have a lot of choices so that we can show that the menu scrolls.